src/Configuration/Configuration.php

```diff
@@ -17,87 +17,87 @@
 final class Configuration
 {
     /**
-     * @var bool
+     * @var bool @var bool
      */
     private $isDryRun = false;
 
     /**
-     * @var bool
+     * @var bool @var bool
      */
     private $showProgressBar = true;
 
     /**
-     * @var bool
+     * @var bool @var bool
      */
     private $areAnyPhpRectorsLoaded = false;
 
     /**
-     * @var bool
+     * @var bool @var bool
      */
     private $mustMatchGitDiff = false;
 
     /**
-     * @var bool
+     * @var bool @var bool
      */
     private $shouldClearCache = false;
 
     /**
-     * @var string
+     * @var string @var string
```

Comment:
@TomasVotruba it seems weird, try running rectify locally and working ok: 

```bash
➜  rector git:(enable-move-variable-dec) bin/rector rectify --config rector-ci.php src/Configuration/Configuration.php --dry-run --clear-cache 
Rector dev-master@bd52969
Config file: rector-ci.php

 4/4 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

                                                                                                                        
 [OK] Rector is done!                                                                                                   
                                                                                                                        
```

Reply:
@TomasVotruba I tried disable again the rule and it keep adding `@var` via Rector Rectify auto commit, see https://github.com/rectorphp/rector/pull/4659/commits/4408426fdd128dd5097c971c698a528fc70219c8 , do you have idea?

Reply:
What ECS rule is causing this?

Reply:
I don't know, I tried disable ecs check command and it keep duplicating, see https://github.com/rectorphp/rector/pull/4659/commits/46e051199d035a35351cc071a7ebfa1002dfa6a7